django_freeradius/templates/django_freeradius/prefix_pdf.html

```diff
@@ -1,10 +1,6 @@
-<!DOCTYPE html>{% load static %}
+<!DOCTYPE html>
 <html>
-<head>
-    {% block style %}
-	<link href="{% static 'django-freeradius/css/freeradius.css' %}" rel="stylesheet" />
-    {% endblock %}
-</head>
+<head></head>
```

Comment:
replace the contents of this file with the following:

```html
<!DOCTYPE html>{% load i18n %}
<html>
<head>
    <title>{% trans 'WiFi username and passwords' %}</title>
    <style type="text/css">
        body {
            font-size: 15px;
            font-family: sans-serif
        }
        table { width: 100%; border-collapse: collapse }
        table th { font-weight: bold }
        table th, table td {
            border: 1px solid #ccc;
            padding: 10px;
        }
    </style>
</head>
<body>
<table>
    <tr>
        <th>{% trans 'Username' %}</th>
        <th>{% trans 'Password' %}</th>
    </tr>
    {% for row in users %}
    <tr>
	{% for cell in row %}
        <td>{{ cell }}</td>
	{% endfor %}
    </tr>
    {% endfor %}
</table>
</body>
</html>
```

Users can supply a different template by using the `DJANGO_FREERADIUS_BATCH_PDF_TEMPLATE` setting.